datadog/resource_datadog_security_monitoring_rule.go

```diff
@@ -1433,3 +1443,22 @@ func resourceDatadogSecurityMonitoringRuleDelete(ctx context.Context, d *schema.
 
 	return nil
 }
+
+func resourceDatadogSecurityMonitoringRuleCustomizeDiff(ctx context.Context, diff *schema.ResourceDiff, meta interface{}) error {
+	if validate, ok := diff.GetOkExists("validate"); !ok || (ok && !validate.(bool)) {
+		// Explicitly skip validation
+		log.Printf("[DEBUG] Validate is %v, skipping validation", validate.(bool))
+		return nil
+	}
+
+	providerConf := meta.(*ProviderConfiguration)
+	apiInstances := providerConf.DatadogApiInstances
+	auth := providerConf.Auth
+
+	if payload, err := buildCreatePayload(diff); err == nil {
```

Comment:
What should we do in case `err` is not `nil`?

Reply:
Sure, lets log it as Debug.